NitroxClient/GameLogic/PlayerLogic/RemotePlayerIdentifier.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+
+namespace NitroxClient.GameLogic.PlayerLogic;
+
+/// <summary>
+/// Attached to a RemotePlayer. Useful to determine that this script's GameObject is in the root of a RemotePlayer.
+/// </summary>
```

Comment:
Might be good to explain what a root object is, for new devs especially.
```suggestion
/// <summary>
/// Attached to a RemotePlayer. Useful to determine that this script's GameObject is in the root of a RemotePlayer.
/// </summary>
/// <remarks>
/// The root of an object is defined as the top most GameObject as when an object hierarchy first was spawned in. Either from a prefab, or in Nitrox' case, a cloned root game object.
/// </remarks>
```